MoA/attention/cache_utils.py

```diff
@@ -14,7 +14,7 @@ def __init__(
         band_size: List[List[int]],
         pattern_index: List[List[int]],
     ) -> None:
-        self.seen_tokens = 0
+        self.cache_position = 0
```

Comment:
Is this change due to the interface change in Transformers? If so, it's perfectly fine.

Reply:
Yes. The seen_tokens attribute is deprecated and was removed in v4.41.